tests/integration/test_start_available_mip_dna.py

```diff
@@ -0,0 +1,319 @@
+        },
+    ).respond_with_json(
+        {
+            "id": "1",
+            "logged_at": "",
+            "started_at": "",
+            "completed_at": "",
+            "out_dir": "out/dir",
+            "config_path": "config/path",
+        }
+    )
+
+    # WHEN running mip-dna start-available
+    result: Result = cli_runner.invoke(
+        base,
+        [
+            "--config",
+            config_path.as_posix(),
+            "workflow",
+            "mip-dna",
+            "start-available",
+        ],
+    )
+
+    # THEN a scout command is called to export panel beds
+    subprocess_mock.run.assert_any_call(
+        [
+            f"{test_root_dir}/scout/binary",
+            "--config",
+            f"{test_root_dir}/scout/config",
+            "export",
+            "panel",
+            "--bed",
+            ANY,
+            ANY,
+            ANY,
+            "--build",
+            "37",
+        ],
+        check=False,
+        stdout=ANY,
+        stderr=ANY,
+    )
+
+    # The order of the bed arguments is not deterministic, so we need to look at them as a set
+    # this will be fixed so that the order is always the same in the new implementation of
+    # starting MIP-DNA pipelines
+    bed_args = subprocess_mock.run.call_args_list[0][0][0][6:9]
+    assert set(bed_args) == {
+        GenePanelMasterList.OMIM_AUTO,
+        GenePanelMasterList.PANELAPP_GREEN,
+        "panel_test",
+    }
+
+    # THEN a scout command is called to export managed variants
+    subprocess_mock.run.assert_any_call(
+        [
+            f"{test_root_dir}/scout/binary",
+            "--config",
+            f"{test_root_dir}/scout/config",
+            "export",
+            "managed",
+            "--build",
+            "37",
+        ],
+        check=False,
+        stdout=ANY,
+        stderr=ANY,
+    )
+
+    # THEN a mip-dna analysis is started with the expected parameters
+    subprocess_mock.run.assert_any_call(
+        f"{mip_dna_path}/conda_bin run --name S_mip12.1 "
+        f"{mip_dna_path}/bin analyse rd_dna --config {mip_dna_path}/config/mip12.1-dna-stage.yaml "
+        f"{case.internal_id} --slurm_quality_of_service normal --email testuser@scilifelab.se",
+        check=False,
+        shell=True,
+        stdout=ANY,
+        stderr=ANY,
+    )
+
+    # THEN a successful exit code is returned
+    assert result.exit_code == 0
+
+    # THEN an analysis has been created for the case
+    assert len(case.analyses) == 1
+
+    # THEN the case action is set to running
+    status_db.session.refresh(case)
+    assert case.action == CaseActions.RUNNING
+
+    # THEN gene_panels and managed_variant files has been created
+    case_dir = Path(test_root_dir, "mip-dna", "cases", case.internal_id)
+    assert Path(case_dir, "gene_panels.bed").exists()
+    assert Path(case_dir, "managed_variants.vcf").exists()
+    assert Path(case_dir, "pedigree.yaml").exists()
+
```

Comment:
is it possible to check anything in Trailblazer? Maybe that there is an analysis for the case? It would be nice to cross-check it with the trailblazer id of the analysis entry in statusdb